src/index.js

```diff
@@ -35,6 +36,10 @@ export default class SaucelabsConnector {
             retries:    WEB_DRIVER_CONFIGURATION_RETRIES,
             timeout:    WEB_DRIVER_CONFIGURATION_TIMEOUT
         });
+
+        this.options = {
+            showBrowserStartMessage: options.showBrowserStartMessage

```

Comment:
What's about extra options, I need access to `--direct-domains` and `--no-ssl-bump-domains` flags of `sauce-tunnel`. Also, I think it would be very useful if `direct-domains` options will have the default value, that contains at least Google domains `*.google.com,*.gstatic.com,*.googleapis.com`, otherwise browsers on some Android configurations can not be started due to failed certificates.